docs/body_file.md

```diff
@@ -8,138 +8,156 @@ tags:
-
-Corresponding MFT entry:
-
-`MFT Entry Header Values:`
-`Entry: 11        Sequence: 11`
-`$LogFile Sequence Number: 1070011`
-`Allocated Directory`
-`Links: 1`
-``
-`$STANDARD_INFORMATION Attribute Values:`
-`Flags: Hidden, System`
-`Owner ID: 0`
-`Security ID: 257  ()`
-`Created:    2020-02-01 10:48:44.857384500 (CET)`
-`File Modified:  2020-02-01 10:48:44.857384500 (CET)`
-`MFT Modified:   2020-02-01 10:48:44.857384500 (CET)`
-`Accessed:   2020-02-01 10:48:44.857384500 (CET)`
-``
-`$FILE_NAME Attribute Values:`
-`Flags: Directory, Hidden, System`
-`Name: $Extend`
-`Parent MFT Entry: 5     Sequence: 5`
-`Allocated Size: 0       Actual Size: 0`
-`Created:    2020-02-01 10:48:44.857384500 (CET)`
-`File Modified:  2020-02-01 10:48:44.857384500 (CET)`
-`MFT Modified:   2020-02-01 10:48:44.857384500 (CET)`
-`Accessed:   2020-02-01 10:48:44.857384500 (CET)`
-``
-`Attributes: `
-`Type: $STANDARD_INFORMATION (16-0)   Name: N/A   Resident   size: 72`
-`Type: $FILE_NAME (48-3)   Name: N/A   Resident   size: 80`
-`Type: $INDEX_ROOT (144-4)   Name: $I30   Resident   size: 552`
+---
+
+## Known Issues
+<!-- Issues have been merged from multiple files. Also, issues may apply to The Sleuth Kit more than body files - must reconcile. 
+NOTE: Nov/Dec 2022, validate if each is still an issue -->
+Known shortcomings with body file format are:
+
+* Undocumented granularity of timestamp, current implementation by
+  [The Sleuth Kit](sleuthkit.md) appears to be seconds. See [here](https://github.com/sleuthkit/sleuthkit/issues/1810).
+* Undocumented extended file mode of `-/-rwxrwxrwx`.
+* Undocumented and inconsistent application of TSK metadata addresses. See [here](https://github.com/sleuthkit/sleuthkit/issues/1809)
+* Undocumented and inconsistent application of owner UID. See here.[here](https://github.com/sleuthkit/sleuthkit/issues/1830)
+* Date and time values do not indicate a time zone or if daylight savings applies. All timestamps are assumed to be in UTC.
+* Body file encoding is not specified, UTF-8 is assumed.
+* The name field can contain (`$FILE_NAME`) to indicate the bodyfile entry was derived from a NTFS `$FILE_NAME` attribute instead of `$STANDARD_INFORMATION` and `$DATA` attributes. Note that the exact behavior is not documented by the SleuthKit project.
+* The `name` field can contain `-\> symbolic_link_target` but `fls` does not appear to support this for NTFS. Also see [here](https://github.com/sleuthkit/sleuthkit/issues/2645).
+* It is unclear if the symbolic link target can be used in combination with the `$FILE_NAME` suffix.
+* It is unclear which characters should be escaped, by observation `|`
+  and `\` are both escaped with `\` in the name field by `fls` but `mactime`
+  is unable to handle a name that contains the `|` character. See [here](https://github.com/sleuthkit/sleuthkit/issues/2124). _Note: Other implementations are known to not escape `\`._
```

Comment:
consistency nit: `_Note` on this line is italic but not on other lines

Reply:
Removed as "Note", and instead made a major bullet.